lib/rex/powershell/script.rb

```diff
@@ -32,7 +33,7 @@ class Script
 
     def initialize(code)
       @code = ''
-      @rig = Rex::RandomIdentifier::Generator.new
+      @rig = Rex::RandomIdentifierGenerator.new
```

Comment:
the diff reverted this change which is necessary, can you please put this back to Rex::RandomIdentifier::Generator , thanks